tests/test_builtins.py

```diff
@@ -453,6 +459,243 @@ def test_setdefault(self):
 
 class TestList(TestPenty):
 
+    def test_add(self):
+        self.assertIsType(
+            'x + y', pentyping.List[{int}],
+            env={'x': pentyping.List[{int}],
+                 'y': pentyping.List[{int}]}
+        )
+        self.assertIsType(
+            'x + y', pentyping.List[{int, float}],
+            env={'x': pentyping.List[{int}],
+                 'y': pentyping.List[{float}]}
+        )
+        with self.assertRaises(TypeError):
```

Comment:
I haven't been testing type errors, you're right I should have done that too :-)